pandas/tests/indexes/categorical/test_constructors.py

```diff
@@ -12,10 +12,14 @@
 
 class TestCategoricalIndexConstructors:
     def test_construction_disallows_scalar(self):
-        msg = "must be called with a collection of some kind"
-        with pytest.raises(TypeError, match=msg):
+        with tm.assert_produces_warning(FutureWarning, match="without passing data"):
             CategoricalIndex(categories=list("abcd"), ordered=False)
```

Comment:
in #38614 some tests were changed. so I guess we should also be checking the result as well as that a warning is raised.